packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -146,7 +146,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     return (
       <div ref='root' className={ css('ms-Dropdown-container') }>
         { label && (
-          <Label className={ css('ms-Dropdown-label') } id={ id + '-label' } ref={ this._resolveRef('_dropdownLabel') } required={ required }>{ label }</Label>
+          <Label className={ css('ms-Dropdown-label') } id={ id + '-label' } htmlFor={ id } ref={ this._resolveRef('_dropdownLabel') } required={ required }>{ label }</Label>
```

Comment:
this should happen only if `ariaLabel` option is not set, Also, with this change we don't need to set `aria-label = { label }` on main dropdown div as that will be redundant.
```
<Label htmlFor={ ariaLabel ? undefined: id } />
...
// Dropdown
<div 
   role='combobox'
   aria-label={ ariaLabel }
   .... />